src/plugins/scopes/ScopesPlugin.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+
+import BasePlugin from '../BasePlugin';
+import ScopesApi from './ScopesApi';
+import type Program from '../../elements/types/Program';
+import type Node from '../../elements/Node';
+
+export default class ScopesPlugin extends BasePlugin {
```

Comment:
I'm saying it is not a common practise, as far as i'm aware, to create empty classes with no logic to extend it afterwards, so for me, as a developer interested to write a plugin, wouldn't be a lot of sense to extend from that plugin.

At the moment at least.